gensim/models/ldamodel.py

```diff
@@ -977,7 +985,9 @@ def __getitem__(self, bow, eps=None):
         Ignore topics with very low probability (below `eps`).
 
         """
-        return self.get_document_topics(bow, eps)
+        #Is eps equivalent to minimum_probability?
+        eps = self.minimum_probability
```

Comment:
what is the reason to add this line? it is used anyway in https://github.com/parthoiiitm/gensim/blob/2ac2f951452e7d968acace2b6d7b11be8b79ad62/gensim/models/ldamodel.py#L912